File: src/main/webapp/js/app/transaction/paket.service.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


(function() {

	var services = angular.module('je-transaction-paket-service', [ 'ngResource' ]);

	
	services.factory('PaketListFactory', [ '$resource', function($resource) {

		return $resource('service/transaksi/paket', {}, {
			query : {
				method : 'GET',
				isArray : false
			},
			create : {
				method : 'POST'
			}
		});

	} ]);
	
	services.factory('PaketEditFactory', [ '$resource', function($resource) {

		return $resource('service/transaksi/paket/:id', {}, {
			show : {
				method : 'GET',
				params: {id: '@id'}
			}
//			update : {
//				method : 'PUT',
//				params :{id: '@id'}
//			}
		});

	} ]);
	
})();

File: src/main/java/com/je/spring/rest/master/service/PaketService.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.je.spring.rest.master.service;

import com.je.spring.rest.master.model.Paket;
import java.util.List;
/**
 *
 * @author Prasetyo
 */
public interface PaketService {
    
    public List<Paket> getAll();
    public Paket getById(int id);
    public long count();
    public void insert(Paket paket);
    
}
